lib/run/summary.js

```diff
@@ -31,11 +31,14 @@ proxyPropertyList = function (options, name) {
          * @returns {Object} [A JSON represented list of properties]
          */
         toJSON: function () {
-            return {
-                values: _.map(_.get(options, name, {}), function (value, key) {
-                    return { key: key, name: key, value: value, type: 'string' };
+            var exportOption = _.get(options, name);
```

Comment:
Weird this variable is not export option. Just feeling a bit odd that it isn't named something like `obj`. Also it should have default {} to avoid erroring out.